### DISCUSSION TITLE
招募filecoin技术负责人

### DISCUSSION THREAD

**Discussion by icedtea6** [filecoin-project/community-china#27] [人才招聘]
我们在招募做filecoin挖矿的技术，懂硬件懂系统挖矿优化和有经验的技术 欢迎联系 1021979490@qq.com

**Comment by changlizhi** [comment]
成都招filecoin运维15k+开发20k+，管中午饭，租房补贴一半。欢迎联系419724189@qq.com

**Comment by icedtea6** [comment]
![Uploading image.png…]()

**Comment by ljh8mg1018** [comment]
有沒有技術會的!台灣!!或遠程作業管理問題如下!!高手幫幫忙信箱ljh8mg@gmail.com
![2](https://user-images.githubusercontent.com/84759516/129132437-6a77eab1-2123-47ce-a20c-b4a53a0cb9ff.jpg)
![1628733458189](https://user-images.githubusercontent.com/84759516/129132440-2be9c1e5-f000-49c4-855e-7794cf486da9.jpg)
